src/librustc_data_structures/profiling.rs

```diff
@@ -145,6 +148,7 @@ const EVENT_FILTERS_BY_NAME: &[(&str, EventFilter)] = &[
     ("query-blocked", EventFilter::QUERY_BLOCKED),
     ("incr-cache-load", EventFilter::INCR_CACHE_LOADS),
     ("query-keys", EventFilter::QUERY_KEYS),
+    ("llvm", EventFilter::LLVM),
```

Comment:
When you're ready to merge, don't forget to update the `-Z help` docs 🙂

Reply:
waited with that until the PR that I build on to is merged so I can rebase and get your changes :) but leaving this as unresolved so I do not forget.